### DISCUSSION TITLE
envd slogan

### DISCUSSION THREAD

**Discussion by gaocegege** [tensorchord/envd#834] [General]
Our description now is `Development environment for machine learning`

It does not show readers the benefits of envd, we need a better slogan before v0.3.0.

**Comment by gaocegege** [comment]
Docker alternative for AI/ML applications.

**Comment by popfido** [comment]
`A better way for building containerized AI/ML applications and projects` ? 

Since `best` is blocked by the Advertisement law of Mainland China.

**Reply by gaocegege** [reply]
A better way for building containerized AI/ML applications

LGTM

**Reply by kemingy** [reply]
A better way to build and deliver containerized AI/ML applications

**Comment by zwpaper** [comment]
1. build AI, like AI, in containers.
2. build AI containers, like AI applications.

a slogan should be short and refined, so maybe we should drop `/ML` in `AL/ML`, and building AI is enough to indicate building AI applications.

we use python to build AI containers, is expecting this to be the way users use in AI applications, 
so by using envd, users can build AI containers like they are building AI applications.

**Comment by gaocegege** [comment]
devkit for AI/ML applications

**Comment by gaocegege** [comment]
The missing piece of AI infrastructure

**Reply by popfido** [reply]
Can't hold myself from laughing. It's like a  **"Lord of the Rings" for your AI project**.

**Reply by gaocegege** [reply]
The Last of the Jedi for your AI projects.

**Comment by gaocegege** [comment]
Machine learning environment made easy

**Comment by TaylorHere** [comment]
Building your AI/ML applications
